src/HubphAPI.php

```diff
@@ -73,7 +73,7 @@ public function prGet($org, $project, $id)
         return $this->gitHubAPI()->api('pull_request')->show($org, $project, $id);
     }
 
-    public function prCreate($org, $project, $title, $body, $base, $head)
+    public function prCreate($org, $project, $title, $body, $base, $head, &$response = null)
```

Comment:
Let's make a new method that returns the response, and make `prCreate` call through to the new function, discard the result and return `$this` as it currently does.

Reply:
Maybe name the new method `prOpen()`?

Reply:
Done